src/components/victory-scatter/victory-scatter.jsx

```diff
@@ -253,7 +272,37 @@ export default class VictoryScatter extends React.Component {
     return Helpers.evaluateStyle(baseDataStyle, data);
   }
 
-  renderPoint(data, index, calculatedProps) {
+  static getKeyedData(data) {
+    return data.reduce((keyedData, datum, idx) => {
```

Comment:
And for context, when I started, these methods were more VictoryScatter-specific.  But no reason why they can't be pulled out now.